extensions/blocks/contact-form/components/jetpack-contact-form.js

```diff
@@ -4,10 +4,19 @@
 import classnames from 'classnames';
 import emailValidator from 'email-validator';
 import { __, sprintf } from '@wordpress/i18n';
-import { Button, PanelBody, Path, Placeholder, TextControl } from '@wordpress/components';
+import {
+	BaseControl,
+	Button,
+	PanelBody,
+	Path,
+	Placeholder,
+	SelectControl,
+	TextareaControl,
+	TextControl,
+} from '@wordpress/components';
 import { Component, Fragment } from '@wordpress/element';
 import { compose, withInstanceId } from '@wordpress/compose';
-import { InnerBlocks, InspectorControls } from '@wordpress/editor';
+import { InnerBlocks, InspectorControls, URLInput } from '@wordpress/editor';
```

Comment:
I think we are supposed to use the one from `@wordpress/block-editor` now

Reply:
That'll require some package upgrades: these things were moved to `@wordpress/block-editor` in `@wordpress/editor` version 9.4.0, Jetpack currently imports version 8.0.0.

It probably should be done as part of a broader upgrade of the packages Jetpack uses.

Reply:
We can't use `@wordpress/block-editor` just yet, unfortunately. It was added to WordPress in version 5.2:
https://core.trac.wordpress.org/ticket/46429

Jetpack still supports WordPress 5.1 for now. We'll most likely update the minimum version supported by Jetpack once WordPress 5.3 has been released, since we've tried to stick to support current version -1 so far. You can read more about this in #13620.